package.json

```diff
@@ -31,11 +32,13 @@
     "mongoose": "^6.9.1",
     "morgan": "^1.10.0",
     "pg": "^8.9.0",
+    "postman-request": "^2.88.1-postman.32",
     "stream": "^0.0.2"
   },
   "devDependencies": {
     "eslint": "^8.34.0",
     "eslint-config-airbnb": "^19.0.4",
+    "jest": "^29.4.3",
```

Comment:
do you suppose use postman-request library with jest?

Reply:
yepp i used postman-request with jest

Reply:
nice awesome!